examples/step-22/doc/intro.dox

```diff
@@ -857,11 +857,11 @@ This functions obviates, by the way, also the call to the
 
 <h4>Performance optimizations</h4>
 
-The program developed below has seen a lot of TLC. We have run it over and
-over under profiling tools (mainly <a
+The program developed below has seen a lot of total lines of code (TLC).
```

Comment:
```suggestion
The program developed below has seen a lot of tender loving care.
```
🙂

Reply:
Done. I honestly believed @masterleinad was just joking.

Reply:
No, this is what TLC is commonly understood to mean :-)